tests/one_taskset.rs

```diff
@@ -41,14 +42,33 @@ fn test_single_taskset() {
     config.users = Some(2);
     config.hatch_rate = 4;
     config.status_codes = true;
+
+    let mut paths_and_weights = vec![(INDEX_PATH, 9), (ABOUT_PATH, 3)];
+    let mut taskset = GooseTaskSet::new("LoadTest");
+
+    while let Some(item) = paths_and_weights.pop() {
+        let path = item.0;
+        let weight = item.1;
+
+        let closure: GooseTaskFunction = Arc::new(move |user| {
+            Box::pin(async move {
+                let _goose = user.get(path).await?;
+                Ok(())
+            })
+        });
+
+        let task = GooseTask::new(closure).set_weight(weight).unwrap();
+        // We need to do the variable dance as taskset.register_task returns self and hence moves
+        // self out of `taskset`. By storing it in a new local variable and then moving it over
+        // we can avoid that error.
+        let new_taskset = taskset.register_task(task);
+        taskset = new_taskset;
+    }
+
     let goose_stats = crate::GooseAttack::initialize_with_config(config.clone())
         .setup()
         .unwrap()
-        .register_taskset(
-            taskset!("LoadTest")
-                .register_task(task!(get_index).set_weight(9).unwrap())
-                .register_task(task!(get_about).set_weight(3).unwrap()),
-        )
+        .register_taskset(taskset)
```

Comment:
As noted above, this removes an existing test and replaces it with a new one. Instead it should leave the old existing test, and add a new one in the same file with a different name, so we're testing multiple code paths.

Reply:
Yep - sounds good.

I will copy the test.

Reply:
done